posts/2019-09-18-upcoming-docsrs-changes.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Upcoming docs.rs changes"
+author: The Rust Infrastructure Team
+---
+
+On September 30nd breaking changes will be deployed to the [docs.rs] build
+environment. [docs.rs] is a free service building and hosting documentation for
+all the crates published on [crates.io]. It's [open source][docsrs-source] and
+maintained by the [Rustdoc team][rustdoc-team].
+
+## What will change
+
+Builds will be executed inside the [rustops/crates-build-env] Docker image.
+That image contains a lot of system dependencies installed to ensure we can
+build as many crates as possible. It's already used by [Crater], and we added
```

Comment:
It'd be nice to tell users that they can provide documentation without requiring dependencies (that's what we do in gtk-rs for example). It'd prevent to have build failures.

Reply:
Do you have links with any example of that?

Reply:
Sure!

We disable the linkage in the `build.rs` [file](https://github.com/gtk-rs/sys/blob/master/gdk-sys/build.rs) and we use the provided `docs.rs` feature in the `Cargo.toml` [file](https://github.com/gtk-rs/sys/blob/master/gdk-sys/Cargo.toml#L52) to enable/disable features too.